addons/editor-cleanup-plus/addon.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Enhanced clean up tool",
+  "description": "Enhances the clean up tool with smart column-by-column block organisation and automatic removal of unused lists/variables.",
+  "credits": [
+    {
+      "name": "griffpatch"
+    },
+    {
+      "name": "Chrome_Cat",
+      "link": "https://scratch.mit.edu/users/Chrome_Cat/"
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true,
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "versionAdded": "1.39.0",
+  "tags": ["editor", "codeEditor", "recommended"],
+  "enabledByDefault": false
```

Comment:
Shouldn't this be the other way around? Cleanup+ enabled by default, devtools disabled?

Reply:
On issue #6671 I think I explained some reasons in favor of not enabling cleanup+ by default

Reply:
I agree with unused variable deletion being an off by default setting. I don't see a reason why devtools should stay enabled by default after this though.